exercises/practice/wordy/.docs/instructions.append.md

```diff
@@ -0,0 +1,5 @@
+# Instructions append
+
+## Zig-specific instructions
+
+If an error occurs, return `null`.
```

Comment:
Why not return an exception that (hopefully) provides a directive message on how to avoid raising that exception instead?  This is a practice exercise, and as a library should not swallow exceptions.